course/templates/course/home.html

```diff
@@ -2,16 +2,18 @@
 {% load i18n %}
 
 {% block content %}
+  {% block cover %}
   <div class="jumbotron">
-    {% trans "RELATE" as RELATE %}
-    <h1>{% blocktrans %} Welcome to {{ RELATE }} {% endblocktrans %}</h1>
+    <h1>{% blocktrans %} Welcome to {{ relate_site_name }} {% endblocktrans %}</h1>
     <p>{% trans "RELATE is an Environment for Learning And TEaching" %}</p>
```

Comment:
Did you want to change this, too?

Reply:
I don't think so. Because that's an abbr. If people want to replace that, they should do by template overriding.

Reply:
Fair point.